telehealth-example/src/main/java/com/pubnub/components/example/getting_started/ui/view/Login.kt

```diff
@@ -0,0 +1,209 @@
+                            if (member != null) {
+                                visible = false
+                                openChannelActivity(context, member.id, member.type)
+                            } else {
+                                visible = true
+                            }
+                        },
+                        shape = RoundedCornerShape(20),
+                        colors = ButtonDefaults.buttonColors(
+                            backgroundColor = Color.hsl(
+                                196F,
+                                0.65F,
+                                0.57F,
+                                1F
+                            )
+                        )
+                    ) {
+                        Text(
+                            text = stringResource(R.string.login),
+                            style = TextStyle(
+                                fontSize = 16.sp,
+                                color = Color.White,
+                                fontWeight = FontWeight.Bold
+                            )
+                        )
+                    }
+                }
+                AnimatedVisibility(
+                    visible = visible,
+                    enter = fadeIn(
+                        initialAlpha = 0.4f
+                    ),
+                    exit = fadeOut(
+                        animationSpec = tween(durationMillis = 250)
+                    )
+                ) {
+                    Row(modifier = Modifier.padding(top = 12.dp, start = 24.dp)) {
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_icon),
+                            contentDescription = "error icon",
+                            modifier = Modifier.padding(top = 4.dp)
+                        )
+                        Text(
+                            text = stringResource(R.string.password_error),
+                            style = TextStyle(
+                                fontSize = 16.sp,
+                                color = Color.hsl(2F, 0.72F, 0.53F, 1F)
+                            ),
+                            modifier = Modifier.padding(start = 8.dp)
+                        )
+                    }
+                }
+
+            }
+            Box(
+                modifier = Modifier
+                    .background(Color.hsl(0F, 0F, 0.96F, 1F))
+                    .align(
+                        Alignment.BottomCenter
+                    )
+            ) {
+                HyperlinkText(
+                    modifier = Modifier
+                        .padding(top = 40.dp, start = 72.dp, end = 72.dp, bottom = 30.dp),
+                    fullText = stringResource(R.string.password_info),
+                    linkText = listOf("Demo page")
+                )
+            }
+        }
+    }
+
+    private fun openChannelActivity(packageContext: Context, uuid: String, type: String) {
```

Comment:
Do we need to change the activity? Maybe it'll be better to use NavHost for it and just switch screens?

Reply:
seperate ticket

Reply:
https://pubnub.atlassian.net/browse/UI-1534